### PR TITLE
What does the r above my head mean?

### PR DESCRIPTION
**Pull request by N3D6**
# Document the changes in your pull request

Funny new brain trauma, "What dose r above my hed mean?"

# Changelog

:cl:  
tweak: new brain trauma line
/:cl:

### CODE REVIEW

**Comment by N3D6** [comment]
how the fuck are the checks failing it's literally one line of code

File: strings/traumas.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
 		"CAN SPACE DRAGONS COMMIT MURDERS?",
 		"ok what? How do I keep getting this beard thing? ANd don't send me to mentor because they won't know",
 		"@pick(semicolon)Who is Gus??? Tell me right now or else I'm about to get ugly"
```
Comment (Moltijoe) [inline strings/traumas.json]:
```suggestion
		"@pick(semicolon)Who is Gus??? Tell me right now or else I'm about to get ugly",
```

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: strings/traumas.json (hunk 1 above)
Comment (N3D6) [reply strings/traumas.json]: fixed

**Comment by Moltijoe** [comment]
not a clue what's causing the bot error

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**